src/devices/machine/k056230.cpp

```diff
@@ -2,7 +2,24 @@
 // copyright-holders:Fabio Priuli
 /***************************************************************************
 
-    Konami IC 056230 (LANC)
+Konami IC 056230 (LANC)
+
+Schematic (located on page 28): http://arcarc.xmission.com/PDF_Arcade_Manuals_and_Schematics/Racing_Force_Schematics.pdf
+
+Device Notes:
+-The custom IC itself
+-64k shared ram
+-LS161 4-bit binary counter
+-PAL(0056787) for racinfrc's sub board and plygonet.cpp
+-PAL(0056787A) for zr107.cpp, gticlub.cpp and thunderh's I/O board
+-HYC2485S RS485 transciever
+
+Todo: nearly everything
+-Have the device fired at any irq, not just the ppc from zr107.cpp and gticlub.cpp
+-What exactly "happens" if an IRQ is fired to this device?
+-Is ram shared between the main cpu and other communicating units?
+-Find and hook the "networking" exchange, some sort of UART serial?
+-Daisy chain support, a stream perhaps?
```

Comment:
"TODO: nearly everything" will do. Don't make weird guesses. What's a daisy chain stream?

Reply:
Was intending on having this device communicate with each other over a socket (most likely bitbanger related) but I don't even have a clue if THAT will work because of the fact this is a full fledged custom.